CHANGELOG.md

```diff
@@ -17,12 +17,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### :rotating_light: Breaking changes
 - ...
 
----
+## [v0.15.0] - 2023-09-02
 
-### :bug: Fixes
-- [#967](https://github.com/reviewdog/reviewdog/pull/967) Fix parsing long lines in diffs #967
-- [#1426](https://github.com/reviewdog/reviewdog/pull/1426) Remove default error level
```

Comment:
#967 and #1426 are removed, but it is no problem.
they have already released in v0.14.2.